src/components/site-card/logs-viewer.tsx

```diff
@@ -18,25 +40,99 @@ export function LogsViewer({ logs, onDismiss }: IProps): JSX.Element {
     []
   )
 
+  console.log(`structuredLogs`, structuredLogs)
+
   return (
     <StyledPanel>
-      <StyledPanelHeader onCloseButtonClick={onDismiss}>Logs</StyledPanelHeader>
-      <RawLogs
-        // RawLogs expect "message" to be a string but in factthey also support elements
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        logItems={logs.map((logEntry) => {
-          return {
-            message: (
-              <span
-                // eslint-disable-next-line @typescript-eslint/naming-convention
-                dangerouslySetInnerHTML={{ __html: parser.toHtml(logEntry) }}
+      <Flex sx={{ flexDirection: `column`, height: `100%` }}>
+        <StyledPanelHeader onCloseButtonClick={onDismiss}>
+          <strong sx={{ color: `blue.80` }}>{siteName}</strong>
+        </StyledPanelHeader>
+        <Tabs sx={{ flexGrow: 1, display: `flex`, flexDirection: `column` }}>
+          <TabList sx={{ flexGrow: 0 }}>
+            <Tab>Logs</Tab>
+            <Tab>Raw logs</Tab>
+          </TabList>
+          <TabPanels sx={{ flexGrow: 1 }}>
+            <TabPanel sx={{ height: `100%` }}>
+              <StyledPanelBodySection>
+                <BuildLogsList
```

Comment:
Couuld the logslist itself scroll, rather than scrolling the whole panel? It means the tabs scroll out of view. The old logs viewer also auto-scrolled the view to the bottom as the logs loaded.